### PR TITLE
Task/gp 773 decode project change status msg

### PR DESCRIPTION
**Pull request by joel-san**
## What does this change?
Provides a code skeleton for decoding the status message for the project_restorer project (i.e [GP-773](https://codemill.atlassian.net/browse/GP-773?atlOrigin=eyJpIjoiY2E0OWNiZGQ1MDc1NDNiNDhmZDViZjI0YTc3NjViYWEiLCJwIjoiaiJ9)).

## How to test
Test with unit tests that the methods are failing correctly.

## Accessibility

- [ ] [Tested with screen reader](https://guardian.github.io/source/?path=/docs/docs-06-accessibility--page#screen-readers)
- [ ] [Navigable with keyboard](https://guardian.github.io/source/?path=/docs/docs-06-accessibility--page#keyboard-navigation)
- [ ] [Colour contrast passed](https://guardian.github.io/source/?path=/docs/docs-06-accessibility--page#colour-contrast)
- [ ] [The change doesn't use only colour to convey meaning](https://guardian.github.io/source/?path=/docs/docs-06-accessibility--page#use-of-colour)

### CODE REVIEW

File: project_restorer/src/main/scala/OwnMessageProcessor.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import com.gu.multimedia.storagetier.framework.MessageProcessorConverters._
```
Comment (fredex42) [inline project_restorer/src/main/scala/OwnMessageProcessor.scala:1]: Think that this file should be called `PlutoCoreMessageProcessor` to keep the naming convention working :)
`OwnMessageProcessor` implies that the messages it's handling are coming from itself

File: project_restorer/src/test/scala/OwnMessageProcessorTest.scala [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          |    "productionOffice": "UK"
+          |  }
+          |""".stripMargin
+
+      val msg = io.circe.parser.parse(msgContent)
+      val emptyJson = Json.fromString("")
+
+      val result = Try{ Await.result(toTest.handleMessage("core.project.update", msg.getOrElse(emptyJson)), 3.seconds)}
+
+      ok("Dummy test so far, but it compiles and runs through the code")
```
Comment (fredex42) [inline project_restorer/src/test/scala/OwnMessageProcessorTest.scala:43]: Can remove this :grin:

**Review by fredex42 — CHANGES_REQUESTED** [review]
:+1: Great! One minor issue but otherwise all good

File: project_restorer/src/test/scala/OwnMessageProcessorTest.scala (hunk 2 above)
Comment (joel-san) [reply project_restorer/src/test/scala/OwnMessageProcessorTest.scala]: Sorry 😬 Fixed now

File: project_restorer/src/main/scala/OwnMessageProcessor.scala (hunk 1 above)
Comment (joel-san) [reply project_restorer/src/main/scala/OwnMessageProcessor.scala]: Fixed :)

**Pull request merged**